app/domain/authentication/authn_jwt/orchestrate_jwt_authentication.rb

```diff
@@ -20,18 +20,18 @@ def call
       private
 
       def authenticate_jwt
-        relevant_vendor = vendor
-        @logger.debug(LogMessages::Authentication::AuthnJwt::ParsingIssuerFromUri.new(relevant_vendor))
+        relevant_authenticator = authenticator_name
+        @logger.debug(LogMessages::Authentication::AuthnJwt::JWTAuthenticatorEntryPoint.new(relevant_authenticator))
 
-        jwt_vendor_configuration = @jwt_configuration_factory.create_jwt_configuration(relevant_vendor)
+        jwt_authenticator_configuration = @jwt_configuration_factory.create_jwt_configuration(relevant_authenticator)
         @jwt_authenticator.call(
-          jwt_configuration: jwt_vendor_configuration,
+          jwt_configuration: jwt_authenticator_configuration,
           authenticator_input: @authenticator_input
```

Comment:
is it ok that its called `authenticator_input` here and later on called `authentication_parameters`?

Reply:
It is not the same, the `authentication_parameters` created by the orchestrator  and contains the `authenticator_input`